public_html/wp-content/mu-plugins/blocks/.eslintrc.js

```diff
@@ -131,5 +131,11 @@ module.exports = {
 		 * @todo `@param` tags should align the variable name and description, just like in PHP.
 		 */
 		'jsdoc/require-returns-description': 'off',
+
+		/*
+		 * This has false positives with every translated string, due to the way `createInterpolateElement()`
+		 * is designed.
+		 */
+		'jsx-a11y/anchor-has-content': 'off',
```

Comment:
I agree that it's not ideal, but it doesn't seem likely that we'd ever create an anchor without text, so I think the likelyhood of this causing tangible problems in practice is very low. It seems much better to me than the hassle and clutter of eslint comments every time we translate a link.

I opened https://github.com/WordPress/gutenberg/issues/21441 in case there are any upstream solutions in the future.

Reply:
One potential workaround that's cleaner would be doing something like this:

```js
a: <a href={ 'https://central.wordcamp.org/contact-us/' } >%s</a>,
```

The `%s` wouldn't be used anywhere, so it's potentially confusing for our future selves, but we could add a `todo` note to go back and remove them when/if WordPress/gutenberg#21441 is resolved.

Reply:
> it doesn't seem likely that we'd ever create an anchor without text

Sure, but the rule is there to catch any mistakes. Having to add the comment every time would also ensure that we _know_ that the interpolated component has content too.

> we could add a todo note …

I don't see how this is that different than adding the eslint comment, and it's more confusing to our future selves.

Reply:
I still think the downsides outweigh the benefits.

I wasn't picturing adding it every time, just on one of them. Or a github issue if you'd prefer.

Reply:
Maybe something like `a: <a href={ 'https://central.wordcamp.org/contact-us/' } >#21441-gutenberg</a>` would be more self-documenting.

Reply:
I don't want us to disable the rule, so as long as adding content into the `<a>` doesn't introduce side effects, we should go with that.

The `%s` style makes more sense to me, since that's basically how printf etc works, otherwise I think future-me would come back and wonder why we're outputting a gutenberg ticket (we're not, but that's what I read first).

And (unsurprisingly) I'd prefer this be surfaced in an issue (blocked by that GB one).

Reply:
I liked `%s` at first for that reason, but then it seemed like it'd be confusing because in the future I might actually try to change it, like making it `%s %s` if I wanted to add another argument to the string.

Linking to the G issue is definitely weird, but I think that's helpful, because it makes it obvious that something odd is happening, and gives a curious dev all of the information they need to figure out why.

Reply:
But that's not how `createInterpolateElement` works, there's no way to do that in practice. It feels much less clear to me to use the github issue inline, but if we're at an impasse otherwise then it's fine.

Reply:
👍 , 6f732edbbe2a0cc2002d37f9ad6778bf97bd2d59